docker/scripts/grub-installer.sh

```diff
@@ -141,7 +141,7 @@ install_grub_osie() {
 			install -Dm755 "$grubefi" "$target/boot/efi/EFI/GRUB2/GRUBAA64.EFI"
 		else
 			# grub-install doesn't error if efibootmgr can't actually set the boot entries/order
-			efibootmgr | tee /dev/stderr | grep -iq grub
+			efibootmgr | tee /dev/stderr | grep -iq ubuntu
```

Comment:
Neat! I've never seen the `tee /dev/stderr` trick. The comment I had was this can be fragile, and break in certain obscure environments (chroots and what not). The native way to do this is in shells (at least those produced after 1993) is:

`efibootmgr 2>&1 | grep -iq ubuntu`

Feel free to ignore this suggestion in preference to putting it into a secondary PR.

Reply:
The `tee` is here so we can see all of `efibootmgr`'s output on stderr while copying to stdout and into `grep`. `efitbootmgr 2>&1` won't duplicate to stderr. This is really just a hack to get around running `efibootmgr` twice, which doesn't really buy us anything really, I'm fine with calling it twice.